src/cmdlib.sh

```diff
@@ -384,6 +415,23 @@ EOF
     else
         rm -vf "${local_overrides_lockfile}"
     fi
+    if [ -e "${configdir}/content_sets.yaml" ]; then
+        create_content_manifest "$configdir"/content_sets.yaml
+        mkdir -p "${tmp_overridesdir}"/contentsetrootfs/usr/share/buildinfo/
+        jq . "${workdir}/tmp/buildinfo/content_manifest.json" > "${tmp_overridesdir}/contentsetrootfs/usr/share/buildinfo/content_manifest.json"
```

Comment:
Could make this clearer by passing the output path to `create_content_manifest` so it directly writes to the rootfs.